types/fluent-ffmpeg/index.d.ts

```diff
@@ -178,7 +178,7 @@ declare namespace Ffmpeg {
         size?: number | undefined;
         bit_rate?: number | undefined;
         probe_score?: number | undefined;
-        tags?: any[] | undefined;
+        tags?: Record<string, string> | undefined;
```

Comment:
@amcasey Sorry, I meant to respond to this. The answer is "I'm not sure". I wanted to try and keep this schema (fairly) open-ended to prevent breaking changes...

Reply:
@robertmain Well, correcting from an array to an object is definitely going to break anyone who was using an array (which seems correct, if that's not what the library expects).  My question was actually whether the type should be something like `Record<string, string | number>` which would be _more_ permissive than what you've got now.

Reply:
@amcasey Technically yes, but if memory serves trying to address that data using an array doesn't actually work in the first place so they're already broken. 

I think I misunderstood your question - I thought you were asking me to update it to have specific properties like 

```ts
tags: {
  year: number,
  artist: string
}
```

I was worried that doing that may break things if ffmpeg returns a tag that isn't enumerated. However, I think your idea of `Record<string, string | number>` makes more sense for (as you said) things like year.

Reply:
Changed in d53f19768397219102e7ea15bbaef8e3b43067d1